pkg/operator/ceph/csi/util.go

```diff
@@ -99,3 +102,20 @@ func templateToDeployment(name, templatePath string, p templateParam) (*apps.Dep
 	}
 	return &ds, nil
 }
+
+func getPortFromENV(env string, defaultPort uint16) uint16 {
+	port := os.Getenv(env)
+	if strings.TrimSpace(port) == "" {
+		return defaultPort
+	}
+	p, err := strconv.ParseUint(port, 10, 64)
+	if err != nil {
+		logger.Errorf("failed to parse port value for env %s. %+v ,using default port %d", env, err, defaultPort)
```

Comment:
It's not a valid error so I think we should "hide" it on a Debugf message.

Reply:
if the user provides some negative value, this will fail do we need to use `Debug` here?

Reply:
If the value is negative, we fallback to the default value and that's enough. So I think Debugf should be used.